proto/buf.gen.legacy.yaml

```
@@ -0,0 +1,5 @@
+version: v1
+plugins:
+  - name: gogofaster
+    out: legacy_protos
+    opt: paths=source_relative
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)